src/v2/view-builder/views/webauthn/ChallengeWebauthnAuthenticatorView.js

```diff
@@ -0,0 +1,142 @@
+import { loc, _, createButton } from 'okta';
```

Comment:
inconsistent naming convention.
- `ChallengeWebauthnAuthenticatorView.js`
- `EnrollWebauthnView.js`

you'd like to either always have 'Authenticator' or none in file name.